paf-mvp-demo-express/src/views/publisher/index.hbs

```diff
@@ -74,7 +66,27 @@
 
         pbjs.que.push(function() {
             pbjs.addAdUnits(adUnits);
-            pbjs.setConfig({debug: true})
+            pbjs.setConfig({
+                debug: true,
+                realTimeData: {
+                    auctionDelay: 1000,
+                    dataProviders: [
+                            {
+                            name: "paf",
+                            waitForIt: true,
+                            params: {proxyHostName: "cmp.pafdemopublisher.com"}
```

Comment:
I am wondering if we need a config object for the PAF library instead of configuring the `proxyHostName` in PrebidJs. When I see this line, I have the feeling that we configure PrebidJS so that it can "configure" PAF call. This dependency seems unnecessary.

Reply:
yes i had the same inclination. if the proxy is set by the PAF lib then we can omit this

Reply:
yes i had the same inclination. if the proxy is set by the PAF lib then we can omit this

Reply:
Excellent point.
We might need to refactor the PAF-lib to define a `config` or `init` method with proxy configuration in it.
I'm thinking we should keep that as a future improvement (even if near future). Otherwise there will be a lot of conflicts.
WDYT?

Reply:
yes we can do this in a new iteration